test/python/pulse/test_transforms.py

```diff
@@ -401,12 +407,286 @@ def test_multiple_schedules(self):
             schedule += Play(SamplePulse([0.0, 0.2]), drive_channel)
             schedules.append(schedule)
 
-        compressed_schedule = compress_pulses(schedules)
+        compressed_schedule = transforms.compress_pulses(schedules)
         original_pulse_ids = get_pulse_ids(schedules)
         compressed_pulse_ids = get_pulse_ids(compressed_schedule)
         self.assertEqual(len(original_pulse_ids), 6)
         self.assertEqual(len(compressed_pulse_ids), 2)
 
 
+class TestAlignSequential(QiskitTestCase):
+    """Test sequential alignment transform."""
+
+    def test_align_sequential(self):
+        """Test sequential alignment without a barrier."""
+        d0 = pulse.DriveChannel(0)
+        d1 = pulse.DriveChannel(1)
+
+        schedule = pulse.Schedule()
+        schedule = schedule.insert(1, instructions.Delay(3, d0), mutate=True)
+        schedule = schedule.insert(4, instructions.Delay(5, d1), mutate=True)
+        schedule = schedule.insert(12, instructions.Delay(7, d0), mutate=True)
+
+        schedule = transforms.remove_directives(transforms.align_sequential(schedule))
+
+        reference = pulse.Schedule()
+        # d0
+        reference.insert(0, instructions.Delay(3, d0), mutate=True)
+        reference.insert(8, instructions.Delay(7, d0), mutate=True)
+        # d1
+        reference.insert(3, instructions.Delay(5, d1), mutate=True)
```

Comment:
It's harder to make comparisons to `schedule` reading this when they're each organized differently. for `sequential`, they should be written sequentially to make it easier to follow
```suggestion
        reference.insert(0, instructions.Delay(3, d0), mutate=True)
        reference.insert(3, instructions.Delay(5, d1), mutate=True)
        reference.insert(8, instructions.Delay(7, d0), mutate=True)
```

Reply:
Fixed in 09d7f8762